attributes/default.rb

```diff
@@ -56,7 +56,7 @@
 
 default['hive2']['conf']['mapreduce_input_size']     = "134217728"
 
-default['hive2']['hudi_version']              = "0.5.0-SNAPSHOT"
+default['hive2']['hudi_version']              = "0.5.1-SNAPSHOT"
```

Comment:
This PR has the same issue as the spark-chef one (https://github.com/logicalclocks/spark-chef/pull/85)

Reply:
And same fix as for the other PR.